mobile/lib/features/welcome/onboarding.dart

```diff
@@ -10,59 +11,60 @@ final themeMode = ValueNotifier(2);
 class CarouselItem {
   final String title;
   final String description;
-  final String imagePath;
-  final double aspectRatio;
+  final Widget imageWidget;
 
-  CarouselItem(this.title, this.description, this.imagePath, this.aspectRatio);
+  CarouselItem(
+    this.title,
+    this.description,
+    this.imageWidget,
+  );
 }
 
 final List<CarouselItem> carouselItems = [
-  CarouselItem("Your keys, your control", "Stay in control of your funds at all time.",
-      "assets/carousel_1.jpg", 30 / 35),
-  CarouselItem("Bitcoin only & Lightning fast.",
-      "The highest level of security, at lightning speed.", "assets/carousel_2.jpg", 30 / 38),
   CarouselItem(
-      "Perpetual futures trading.",
-      "Experience P2P leveraged trading with no counterparty risk.",
-      "assets/carousel_3.jpg",
-      30 / 35),
+      "Welcome Satoshi,",
+      "In 10101, every trade is collateralized and fully self-custodial. \n We use DLCs (Discreet Log Contracts) for this. Your first trade will open a DLC Channel which defines the amount which will be available to trade. ",
+      Image.asset("assets/carousel_1.png")),
+  CarouselItem(
+      "Self-custodial",
+      "The max amount you put at risk will depend on how much you lock up. At the same time, the amount you can win will depend on the amount your channel counterparty will lock up.",
+      const PnlLineChart()),
```

Comment:
I think this widget is great and we should consider reusing it elsewhere in the app!

Reply:
Yea, I was thinking of adding it to the trade screen somewhere